### DISCUSSION TITLE
Variables in Geomap Dynamic GeoJSON (Alpha) GeoJSON URL

### DISCUSSION THREAD

**Discussion by ablokland** [grafana/grafana#60471] [Plugins]
To be able to dynamically filter data based on variables in the dashboard, it would useful that variables can be passed in the GeoJSON URL. This way, a service providing the endpoint can use that information to filter the returned GeoJSON.

I'm willing to invest time into this by implementing the feature and creating a pull-request.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
